From 0909c9f75a78d12359aeaba40039cd1c4e90292f Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Tue, 30 Jul 2019 17:45:53 +0800
Subject: [PATCH] arm64: dts: rockchip: replace earlyprintk with earlycon in
 the bootargs

arm64 do not support earlyprintk, and use earlycon instead.

Change-Id: I089707eb69f6ad985fb4040248dc62d6ccd0dd78
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-android.dtsi    | 2 +-
 arch/arm64/boot/dts/rockchip/rk3328-android.dtsi  | 2 +-
 arch/arm64/boot/dts/rockchip/rk3368-p9-avb.dts    | 2 +-
 arch/arm64/boot/dts/rockchip/rk3368-xikp-avb.dts  | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-android.dtsi b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
index 0671c82a4ece..7e5ab98850d6 100644
--- a/arch/arm64/boot/dts/rockchip/px30-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-android.dtsi
@@ -6,7 +6,7 @@
 
 / {
 	chosen: chosen {
-		bootargs = "earlyprintk=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.veritymode=enforcing androidboot.hardware=rk30board androidboot.console=ttyFIQ0 init=/init kpti=0";
+		bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.veritymode=enforcing androidboot.hardware=rk30board androidboot.console=ttyFIQ0 init=/init kpti=0";
 	};
 
 	fiq-debugger {
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
index e7205336027d..b57f7edebc6b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-android.dtsi
@@ -6,7 +6,7 @@
 
 / {
 	chosen {
-		bootargs = "earlyprintk=uart8250-32bit,0xff130000 swiotlb=1 kpti=0 coherent_pool=1m";
+		bootargs = "earlycon=uart8250,mmio32,0xff130000 swiotlb=1 kpti=0 coherent_pool=1m";
 	};
 
 	fiq-debugger {
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-p9-avb.dts b/arch/arm64/boot/dts/rockchip/rk3368-p9-avb.dts
index 335c7d7cff37..4a3d092382ae 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-p9-avb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-p9-avb.dts
@@ -31,5 +31,5 @@
 };
 
 &chosen {
-	bootargs = "earlyprintk=uart8250,mmio32,0xff690000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.veritymode=enforcing androidboot.hardware=rk30board androidboot.console=ttyFIQ0 androidboot.selinux=permissive init=/init kpti=0";
+	bootargs = "earlycon=uart8250,mmio32,0xff690000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.veritymode=enforcing androidboot.hardware=rk30board androidboot.console=ttyFIQ0 androidboot.selinux=permissive init=/init kpti=0";
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-xikp-avb.dts b/arch/arm64/boot/dts/rockchip/rk3368-xikp-avb.dts
index 5644ad094834..65c01eb1e8c3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-xikp-avb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-xikp-avb.dts
@@ -31,5 +31,5 @@
 };
 
 &chosen {
-	bootargs = "earlyprintk=uart8250,mmio32,0xff690000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.veritymode=enforcing androidboot.hardware=rk30board androidboot.console=ttyFIQ0 androidboot.selinux=permissive init=/init kpti=0";
+	bootargs = "earlycon=uart8250,mmio32,0xff690000 swiotlb=1 console=ttyFIQ0 androidboot.baseband=N/A androidboot.veritymode=enforcing androidboot.hardware=rk30board androidboot.console=ttyFIQ0 androidboot.selinux=permissive init=/init kpti=0";
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
index c27eb1458ca7..0b389d9c1c4c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
@@ -15,7 +15,7 @@
 	compatible = "pine64,rockpro64", "rockchip,rk3399";
 
 	chosen {
-		bootargs = "earlyprintk=uart8250,mmio32,0xff1a0000 swiotlb=1 coherent_pool=1m";
+		bootargs = "earlycon=uart8250,mmio32,0xff1a0000 swiotlb=1 coherent_pool=1m";
 		stdout-path = "serial2:1500000n8";
 	};
 
-- 
2.35.3

